js/ui/pager.js

```diff
@@ -27,12 +28,15 @@ var PAGES_LIMITER = 4,
     PAGER_PAGE_SEPARATOR_CLASS = 'dx-separator',
     PAGER_PAGE_SIZES_CLASS = 'dx-page-sizes',
     PAGER_PAGE_SIZE_CLASS = 'dx-page-size',
+    PAGER_PAGE_SIZE_CLASS_SELECTOR = '.' + PAGER_PAGE_SIZE_CLASS,
     PAGER_NAVIGATE_BUTTON = 'dx-navigate-button',
     PAGER_PREV_BUTTON_CLASS = "dx-prev-button",
     PAGER_NEXT_BUTTON_CLASS = "dx-next-button",
     PAGER_INFO_CLASS = "dx-info",
     PAGER_INFO_TEXT_CLASS = "dx-info-text",
-    PAGER_BUTTON_DISABLE_CLASS = "dx-button-disable";
+    PAGER_BUTTON_DISABLE_CLASS = "dx-button-disable",
+
+    isLegacyKeyboardNavigation = false;
```

Comment:
Why is isLegacyKeyboardNavigation variable is global?

Reply:
Why is not possible to use this.option("useLegacyKeyboardNavigation") in all places where it needs?

Reply:
> Why is isLegacyKeyboardNavigation variable is global?

It is because Page instance don't know about Pager instance where option presents.
I have moved seting the tabindex for the selected page to the another place to get rid of this global variable.